src/cli/onefuzz/api.py

```diff
@@ -53,6 +53,8 @@
 
 UUID_RE = r"^[a-f0-9]{8}-?[a-f0-9]{4}-?[a-f0-9]{4}-?[a-f0-9]{4}-?[a-f0-9]{12}\Z"
 
+CLIENT_SECRET_ENV_VAR = "ONEFUZZ_CLIENT_SECRET"
```

Comment:
just a heads up. Check-pr will probably fail because the client-secret is not set.
You will need to update [this](https://github.com/microsoft/onefuzz/blob/a4357c8fb9b4eb2ceee1206c151e6c027f22a990/src/utils/check-pr/check-pr.py#L141)

Reply:
Ha, yep, just fixed that!

Reply:
Er, nope, something else. But I see what _you_ mean. Fixing.

Reply:
Oh, no, nevermind. We still accept the `--client_secret` arg, so check-pr should still pass as-is.